src/main/java/org/hl7/davinci/priorauth/Database.java

```diff
@@ -253,10 +253,22 @@ public IBaseResource read(String resourceType, Map<String, Object> constraintPar
    * @return the related field of the database
    */
   public String readRelated(String resourceType, Map<String, Object> constraintParams) {
-    logger.info("Database::read(" + resourceType + ", " + constraintParams.toString() + ")");
-    if (resourceType != null && constraintParams != null) {
+    return readString(resourceType, constraintParams, "related");
+  }
+
+  /**
+   * Read the specified column from the database
+   *
+   * @param resourceType     - the FHIR resourceType to read.
+   * @param constraintParams - the search constraints for the SQL query.
+   * @param column           - the column to read.
+   * @return the specified column of the database
+   */
+  public String readString(String resourceType, Map<String, Object> constraintParams, String column) {
```

Comment:
I wasn't sure if readStatus() should call readString() as well. The query it runs is a little bit different. Thoughts on combining them?

Reply:
When I wrote `readStatus` it was expecting only one result. It would actually be better to use the same query as `readString` instead of what it currently is. Go ahead and update it to use `readString`